### PR TITLE
Supermarket Prefix Reservation

### PR DESCRIPTION
**Pull request by coderanger**
A short RFC to document and formalize what I think is already common practice.

### CODE REVIEW

File: new/supermarket-prefix.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The responsible user or users can authorize other users to create cookbooks in
+their prefix, but this requires explicit permission from the prefix holder.
+
+In the short term, prefix management can be handled ad hoc via existing cookbook
+rename/removal tooling in Supermarket. Established prefixes can be added to this
+RFC by pull request.
+
+In the long term, it would be good to see native functionality added to
+Supermarket to enable this, though care will have to be taken at that point to
+ensure abuse is not a problem.
```
Comment (lamont-granquist) [inline new/supermarket-prefix.md]: we might want to specify something like a prefix is at least two letters followed by an underscore, and that when registering a prefix there shall be no cookbooks not owned by the registeree that exist with that prefix or something.

in the case of someone registering a prefix after someone else has already trampled on the prefix that would require some kind of administrative action.

that way nobody can register compat_ (compat_resource) or chef_ (chef_hostname).

**Comment by lamont-granquist** [comment]
:+1: fine with not doing the future thing in this RFC

**Comment by btm** [comment]
as this week's decider, this is accepted contingent on @coderanger adding a little more information on how to contact the supermarket admins in case of any namespace encroachment.

**Comment by adamleff** [comment]
👍

**Comment by adamleff** [comment]
Accepted/assigned RFC 078 in 779cb6f. Thanks!

**Pull request closed without merging**